.changeset/shiny-turkeys-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'graphql-modules': patch
+---
+
+Bind context to async execution avoiding race-conditions
```

Comment:
@enisdenjo do we need to add more info here? users only need to upgrade, right?

Reply:
Just an upgrade is enough. No action from the users needed.